### ISSUE TITLE
Selecting Models

### ISSUE THREAD

**Issue by rlagare** [dotnet/machinelearning-modelbuilder#1999]
**System Information (please complete the following information):**
 - Model Builder 
 - Visual Studio Version (if applicable):

**Describe the bug**
- On which page of the tutorial did you run into an issue (URL): This is not a bug
- Clear description of the problem: Model builder automatically selects the best model for the loaded dataset. It also shows other models that it explored for that particular dataset. In some cases, the models have similar accuracies. 1. Is it possible to consume these other models instead of the one suggested automatically, and how do you do this?

2. Also, how do you inspect the parameters of the trained model? I can only see the training parameters (e.g., learning rate) of the selected model, but not the model parameters that are used to make predictions on testing data.

**To Reproduce**

**Expected behavior**
1. The ability to consume other models that are explored, and not just the model that was automatically suggested by Model builder. 2. The ability to inspect the parameters of the trained model-- this is useful for scientific reporting.

**Screenshots**

**Additional context**

**Comment by torronen** [comment]
@rlagare The temp folder that is mentioned in the output window has all the models as zip files. I have been succesful in copying them and running my own metrics on them with test data. Note that the numbers do not always match exactly with the log output. The params are lost as far as I know, i created an issue about it recently. 

Microsoft team may have some better info.

**Comment by torronen** [comment]
@rlagare LightGBM and FastTree .zip files include the hyperparams inside the zip file, so it might be possible to extract them throuhg .NET interfaces, or worst case, by reading directly from the Model.key file either somehow deserializing the binary file, or even reading as ASCII. Screenshots are from notepad

If we are lucky, we might be able to find the same from all model.zip files.


![kuva](https://user-images.githubusercontent.com/26261427/152658708-e95aab4a-9fe0-41bc-bf44-8c1b1b032e5b.png)

![kuva](https://user-images.githubusercontent.com/26261427/152658587-1dff8b23-0b4f-4ddc-bc30-cbe9507ac61c.png)

**Comment by rlagare** [comment]
Thanks for the feedback @torronen

**Comment by luisquintanilla** [comment]
Updated labels to enhancement / feature request inline with other #1591 work items.
